plugins/woocommerce/src/Internal/Admin/Marketplace.php

```diff
@@ -70,6 +74,25 @@ public static function get_marketplace_pages() {
 		return apply_filters( 'woocommerce_marketplace_menu_items', $marketplace_pages );
 	}
 
+	/**
+	 * Create the menu bubble for extensions menu based on number of updates available.
+	 *
+	 * @return string
+	 */
+	private static function get_marketplace_update_count_html() {
+		$count = WC_Helper_Updater::get_updates_count();
+		if ( empty( $count ) ) {
```

Comment:
I think this `if` statement should go after the second `if` statement below. Otherwise, if no existing updates are available, it doesn't increment the count even for Woo Update Manager